transports/azure-service-bus/message-lock-renewal.md

```diff
@@ -24,6 +24,8 @@ Message lock renewal is only applied on messages actively processed by endpoints
 
 Message lock renewal should be greater than `LockDuration`. When `LockDuration` period is due to expire, Azure Service Bus transport will issue lock renewal request to the broker, keeping message locked and invisible to other consumers. Lock renewal will automatically take place while the total time of the message processing stays less than auto renewal time set by Azure Service Bus transport. The default lock renewal time is 5 minutes. Auto lock renewal will **not** increase `DeliveryCount` of the processed message.
 
+NOTE: message lock renewal is only applied to the currently processed message. Any prefetched messages that are pending processing longer than `LockDuration` time will lose their lock, resulting in `LockLostException` exception when these messages are attempted for completion. To prevent exception when lock renewal is used, it's advised to disable prefetching by setting `PrefetchCount` to zero.
```

Comment:
@SeanFeldman what do you think about this slight change:

Message lock renewal applies to the currently processed message. Prefetched messages not handled within the `LockDuration` time will lose their lock, indicated with a `LockLostException` in the log when they are completed by the transport. To prevent the exception with lock renewal prefetching can be disabled by setting `PrefetchCount` to zero.

Reply:
Maybe we can even remove `when they are completed by the transport`

Reply:
Like:

Message lock renewal applies to the currently processed message. Prefetched messages not handled within the `LockDuration` time will lose their lock, indicated with a `LockLostException` in the log. To prevent the exception with lock renewal prefetching can be disabled by setting `PrefetchCount` to zero.